superx/routes/home.py

```diff
@@ -115,7 +115,10 @@ def add_item():
     cart_list.append(item)
     session['cart'] = cart_list
 
-    return ''
+    was_city_chosen = bool('city' in session)
```

Comment:
FYI - no need for the bool function.
this is the same as 
was_city_chosen = 'city' in session